src/optimise.jl

```diff
@@ -2,11 +2,11 @@
 numeric_type(::IntervalBox{N, T}) where {N, T} = T
 
 """
-    minimise(f, X, structure = SortedVector, tol=1e-3)
+    minimise(f, X, structure = SortedVector, tol=1e-3, f_calls_limit=Inf)
```

Comment:
`Inf` is a `Float64`. Presumably the number of calls should be an integer. So this should be replaced by some large integer like `10^20`, or given a sentinel value like `-1`  (probably preferable).